ngraph_bridge/ngraph_builder.cc

```diff
@@ -4176,6 +4188,13 @@ Status Builder::TranslateGraph(
   //
   ng_function = make_shared<ng::Function>(ng_result_list, ng_parameter_list);
 
+  //
+  // Apply additional passes on the nGraph function here.
+  //
+  ngraph::pass::Manager passes;
+  passes.register_pass<TransposeElimination>();
```

Comment:
Can we rename it to ConsecutiveTransposeElimination which is what it does

Reply:
It actually combines or eliminates consecutive transposes. How about `TransposeFolding` then?